modules/nlu/src/backend/engine2/train-session-service.ts

```diff
@@ -0,0 +1,23 @@
+import * as sdk from 'botpress/sdk'
+
+import { TrainingSession } from '../typings'
+
+const DEFAULT_TRAINING_SESSION: Partial<TrainingSession> = {
+  status: 'idle',
+  progress: 0
+}
+
+function makeTrainSessionKey(language: string): string {
+  return `training:${language}`
+}
```

Comment:
string return value is a bit redundant, could be a one liner
const makeTrainSessionKey = (language: string) => `training:${language}`